src/battle.ts

```diff
@@ -3106,6 +3115,20 @@ class Battle {
 				this.nearSide.active = [null];
 				this.farSide.active = [null];
 				break;
+			case 'multi':
+				this.pokemonControlled = 1;
+				this.p3 = new Side(this, 2);
+				this.p4 = new Side(this, 3);
+				this.p3.foe = this.p4.ally = this.p2;
+				this.p3.ally = this.p4.foe = this.p1;
+				this.p1.ally = this.p3;
+				this.p2.ally = this.p4;
+				this.sides = [this.p1, this.p2, this.p3, this.p4];
+				this.mySide.isFar = this.mySide.ally.isFar = false;
+				this.farSide.isFar = this.farSide.ally.isFar = true;
```

Comment:
This seems wrong, `isFar` should be set by `switchSides` and never changed here.